### PR TITLE
Fix Hadoop spelling/capitalization mistakes

### PR DESCRIPTION
**Pull request by konrad-gerlach**

### CODE REVIEW

File: task-01/task_01.md [hunk 1]
```diff
@@ -195,4 +206,4 @@ Submit the following files packaged into a zip file named `task01.zip`:
 #### Presentation
 
 For each group, one student representative demonstrates their Hadoop setup in action for selected configuration settings. This includes (re)running the MapReduce program and showing and interpreting the effects of the configuration via the Hadoop and Grafana web UIs.  
-Each lab session, we rotate and document the students representing the groups so every student gets her/his turn in the course of the semester.
\ No newline at end of file
+Each lab session, we rotate and document the students representing the groups so every student gets her/his turn in the course of the semester.
```
Comment (tobodner) [inline task-01/task_01.md:209]: I am missing something here, what did you change?

File: task-01/task_01.md [hunk 2]
```diff
@@ -32,6 +32,17 @@ rm 'WordCount\$IntSumReducer.class' 'WordCount\$TokenizerMapper.class' WordCount
...
 
+1b. If you later receive an `Error: Java Heap Space`, then insert the following lines into `/opt/hadoop/etc/hadoop/mapred-site.xml`:
+
+```
+<property>
+    <name>mapreduce.map.java.opts</name>
+    <value>-Xmx4096m</value>
+</property>
+```
+This solution is outlined in the (Stack Overflow Post by User Amar)[https://stackoverflow.com/questions/15609909/error-java-heap-space]
```
Comment (tobodner) [inline task-01/task_01.md:43]: You can directly reference the user's post via https://stackoverflow.com/a/15611829.

**Review by tobodner — CHANGES_REQUESTED** [review]
Thanks for bringing this up! One minor request inline. Then, we are good to go.

File: task-01/task_01.md (hunk 1 above)
Comment (konrad-gerlach) [reply task-01/task_01.md:209]: No change intended there, I don't know.

**Pull request merged**